lib/Doctrine/ODM/PHPCR/UnitOfWork.php

```diff
@@ -2731,6 +2734,9 @@ private function executeMoves($documents)
                 );
             }
 
+            $parentNode = $this->session->getNode(PathHelper::getParentPath($targetPath));
```

Comment:
.. or rather a node. in some cases that node will already have been loaded (f.e. parent mappings *always* load the parent node), I will benchmark to check the performance cost.